.gitignore

```diff
@@ -6,3 +6,17 @@ html/*
 perldoc-browser.conf
 perldoc-browser.sqlite*
 hypnotoad.pid
+
+#Project Files
+.gitignore
```

Comment:
I don't think this is right, .gitignore is maintained in the repository so should not be ignored by git.